inc/tagitem.class.php

```diff
@@ -420,18 +442,26 @@ public static function updateItem(CommonDBTM $item, bool $delete_existing_tags =
         $tag_item = new self();
 
         // Be careful to not check right if the change is coming from the cron
-        if (!Session::isCron() && !$tag::canUpdate()) {
+        if (
+            (!Session::isCron() && !$tag::canUpdate())
+            && !isset($item->input["_plugin_tag_tag_from_rules"])
+        ) {
```

Comment:
Parentheses are not necessary, as there are only ANDs (this is detail)
```suggestion
        if (
            !Session::isCron()
            && !$tag::canUpdate()
            && !isset($item->input["_plugin_tag_tag_from_rules"])
        ) {
```